src/accessibility/RovingTabIndex.tsx

```diff
@@ -131,6 +131,7 @@ export const reducer = (state: IState, action: IAction) => {
         }
```

Comment:
The keyboard navigation still seems broken. On `develop` the `SpaceHierarchy` and `RoomList` appear to work a lot more correctly. But this PR does fix the `MessageActionBar` so far.

Testing strategy:

 - ❌ `SpaceHierarchy`
    - `Arrow` keys appear to navigate inversely up/down
    - ✔ Fixes the the issue of being able to navigate past a space with sub-spaces 
 - ❌ `RoomList`
    - Not able to navigate up and down the room list with `Arrow` keys
 - ✅ `MessageActionBar`
 - ✅ `SpotlightDialog`

Reply:
Argh, somehow potato'd Right & Up being +ve, Down & Left being -ve, getting the verticals wrong.

Reply:
Appreciate you giving those a test!

Reply:
Everything looks good behaviorally 👍 

Testing strategy:

 - ✅ `SpacePanel`/`SpaceHierarchy`
 - ✅ `RoomList`
 - ✅ `MessageActionBar`
 - ✅ `RoomBreadcrumbs`
 - ✅ `SpotlightDialog`